components/config/theme.ts

```diff
@@ -0,0 +1,134 @@
+import { TextStyle } from 'react-native'
+
+export const colors = {
+  dark: {
+    neutral: '#EAEADF',
+    cta: '#398FE4',
+    background: '#13141C',
+    cards: '#161E2C',
+    online: '#0A741C',
+    text: '#aaa',
+    pureBlack: '#000000',
+    pureWhite: '#fff',
+    brandYellow: '#FFDF41',
+    brandPurple: '#923589',
+    purpleDark: '#6C2B68',
+    ctaDark: '#265180',
+    yellow: '#FFDF41',
+    yellowLight: '#FFDF4150',
+    yellowDark: '#FFDF4170',
```

Comment:
@J-Rehan I think these last 3 colors were things I just typed into replit when I was recording the video (yellow, yellowLight, and yellowDark). I didn't intend to actually define those.

Or are those variations you are actually using in components already? 

You can rename any colors in this list as you see fit in order to get a sane list and just sync with Saaniya on the final set of names that you think is good.

Reply:
Those were me, sorry about that. Feel free to drop those, just mirror Figma color pallette.

Reply:
Sure, I will update the color palette once everything is confirmed with Sanya. Let's leave it as it is for now.